ui/components/app/snaps/snap-install-warning/snap-install-warning.js

```diff
@@ -29,118 +41,117 @@ const checkboxStateReducer = produce((state, action) => {
       state[action.checkboxId] = state[action.checkboxId]
         ? !state[action.checkboxId]
         : true;
-
       break;
     default:
       throw new Error(
         'You must provide a type when dispatching an action for checkboxState',
       );
   }
 });
-
 export default function SnapInstallWarning({
   onCancel,
   onSubmit,
   warnings,
   snapName,
+  isOpen,
 }) {
   const t = useI18nContext();
   const [checkboxState, dispatch] = useReducer(checkboxStateReducer, {});
-
   const isAllChecked = warnings.every((warning) => checkboxState[warning.id]);
-
   const onCheckboxClicked = useCallback((checkboxId) => {
     dispatch({ type: 'check', checkboxId });
   }, []);
 
-  const SnapInstallWarningFooter = () => {
-    return (
-      <div className="snap-install-warning__footer">
+  return (
+    <Modal className="snap-install-warning" onClose={onCancel} isOpen={isOpen}>
+      <ModalOverlay />
+      <ModalContent
+        modalDialogProps={{
+          display: Display.Flex,
+          flexDirection: FlexDirection.Column,
+          gap: 4,
+        }}
+      >
+        <ModalHeader
+          onClose={onCancel}
+          childrenWrapperProps={{
+            display: Display.Flex,
+            flexDirection: FlexDirection.Column,
+            alignItems: AlignItems.center,
+            justifyContent: JustifyContent.Center,
+            gap: 4,
+          }}
+        >
+          <AvatarIcon
+            iconName={IconName.Danger}
+            color={IconColor.warningDefault}
+            backgroundColor={BackgroundColor.warningMuted}
+            size={AvatarIconSize.Xl}
+          />
+          <Text variant={TextVariant.headingMd} textAlign={TextAlign.Center}>
+            {t('snapInstallWarningHeading')}
+          </Text>
+        </ModalHeader>
+        <Text textAlign={TextAlign.Center} marginBottom={4}>
+          {warnings.length > 1
+            ? t('snapInstallWarningCheckPlural', [
+                <Text
+                  key="snapNameInWarningDescription"
+                  fontWeight={FontWeight.Medium}
```

Comment:
`fontWeight` should be **regular** for the label except for the snap's name which should be **medium**

Reply:
Right on! I've updated this as well